### PR TITLE
[NT-164] Consume Rewards converted minimum from v1

### PR DESCRIPTION
**Pull request by Scollaco**
# 📲 What 
- Shows Rewards converted minimum based on the value of `convertedMinimum` returned by the API.

# 🤔 Why
- This allows us to have one source of truth and avoids the calculation of conversions to be done on the client side.

# 🛠 How
- Added a new property `convertedMinimum` to our Reward model.
- Changed logic to present the value of that property instead of calculating conversion value based on project conversion rate.
- Updated tests.

# ✅ Acceptance criteria

- [ ] iOS and web should show the same values for conversion minimum on Reward cells.

### CODE REVIEW

File: KsApi/models/Reward.swift [hunk 1]
```diff
@@ -5,6 +5,7 @@ import Runes
 
 public struct Reward {
   public let backersCount: Int?
+  public let convertedMinimum: Double
```
Comment (ifbarrera) [inline KsApi/models/Reward.swift]: Is `convertedMinimum` guaranteed to always be returned? If no conversion is required (ie. project's currency is the same as the user's currency) is the `convertedMinimum` just the same as the `minimum`?

File: KsApi/models/templates/RewardTemplates.swift [hunk 2]
```diff
@@ -24,6 +25,7 @@ extension Reward {
 
   public static let noReward = Reward(
     backersCount: nil,
+    convertedMinimum: 0,
```
Comment (ifbarrera) [inline KsApi/models/templates/RewardTemplates.swift]: I know this is just a template, but I think a `convertedMinimum` would never be 0 🤔 Maybe make this `1` so it's more realistic?

File: KsApi/models/Reward.swift (hunk 1 above)
Comment (Scollaco) [reply KsApi/models/Reward.swift]: Exactly

File: KsApi/models/templates/RewardTemplates.swift (hunk 2 above)
Comment (Scollaco) [reply KsApi/models/templates/RewardTemplates.swift]: I set `convertedMinimum` to  `0` so it matches the `minimum`. For this template, all properties are either `nil` or `0`.

**Pull request merged**